src/component/form-field.js

```diff
@@ -19,6 +21,12 @@ export class FormField {
     this.viewManager = viewManager;
   }
 
+  attached() {
+    if (!existy(this.element.key)) {
```

Comment:
What I need in this case is a `<value> != null`. This is to make sure that anything accept null or undefined is not considered "bad" input. That is why i use `existy(<value>)` instead of !<value>.

Typing `<value> != null` will case a lint error too.